lib/components/form/trimet.styled.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react'
```

Comment:
Why is this file being created?

Reply:
Probably it got in doing a bulk git add, it shouldn't be in here.